packages/design-tokens/src/scss/themes/theme-light/_colors.scss

```diff
@@ -0,0 +1,421 @@
+        border-basic: $custom-02-border-basic,
+        border-subtle: $custom-02-border-subtle,
+        content-basic: $custom-02-content-basic,
+        content-subtle: $custom-02-content-subtle,
+        default: $custom-02-default,
+        hover: $custom-02-hover,
+    ),
+) !default;
+
+$disabled-colors: (
+    background: $disabled-background,
+    border: $disabled-border,
+    foreground: $disabled-foreground,
+    text: $disabled-text,
+) !default;
+
+$emotion-colors: (
+    danger: (
+        active: $emotion-danger-active,
+        background-basic: $emotion-danger-background-basic,
+        background-subtle: $emotion-danger-background-subtle,
+        border-basic: $emotion-danger-border-basic,
+        border-subtle: $emotion-danger-border-subtle,
+        content-basic: $emotion-danger-content-basic,
+        content-subtle: $emotion-danger-content-subtle,
+        hover: $emotion-danger-hover,
+    ),
+    informative: (
+        active: $emotion-informative-active,
+        background-basic: $emotion-informative-background-basic,
+        background-subtle: $emotion-informative-background-subtle,
+        border-basic: $emotion-informative-border-basic,
+        border-subtle: $emotion-informative-border-subtle,
+        content-basic: $emotion-informative-content-basic,
+        content-subtle: $emotion-informative-content-subtle,
+        hover: $emotion-informative-hover,
+    ),
+    success: (
+        active: $emotion-success-active,
+        background-basic: $emotion-success-background-basic,
+        background-subtle: $emotion-success-background-subtle,
+        border-basic: $emotion-success-border-basic,
+        border-subtle: $emotion-success-border-subtle,
+        content-basic: $emotion-success-content-basic,
+        content-subtle: $emotion-success-content-subtle,
+        hover: $emotion-success-hover,
+    ),
+    warning: (
+        active: $emotion-warning-active,
+        background-basic: $emotion-warning-background-basic,
+        background-subtle: $emotion-warning-background-subtle,
+        border-basic: $emotion-warning-border-basic,
+        border-subtle: $emotion-warning-border-subtle,
+        content-basic: $emotion-warning-content-basic,
+        content-subtle: $emotion-warning-content-subtle,
+        hover: $emotion-warning-hover,
+    ),
+) !default;
+
+$focus-colors: (
+    basic: (
+        color-01: $focus-basic-color-01,
+    ),
+) !default;
+
+$gradient-colors: (
+    basic-overlay: (
+        color-01: $gradients-basic-overlay-color-01,
+        color-02: $gradients-basic-overlay-color-02,
+    ),
+) !default;
+
+$neutral-colors: (
+    active: $neutral-active,
+    background-basic: $neutral-background-basic,
+    background-subtle: $neutral-background-subtle,
+    border-basic: $neutral-border-basic,
+    border-subtle: $neutral-border-subtle,
+    content-basic: $neutral-content-basic,
+    content-subtle: $neutral-content-subtle,
+    default: $neutral-default,
```

Comment:
There is no `default` color in emotion tokens. It may (and may not) cause issues once we start generating emotion + neutral color variants for components like `Badge` etc.

Reply:
There is, I forgot to add it 😬